### DISCUSSION TITLE
Syntax for Multiple Index Entries in Sphinx

### DISCUSSION THREAD

**Discussion by tarcine** [sphinx-doc/sphinx#13104] [Q&A]
I would like to know if there is a way to add multiple index entries in Sphinx.

In the tutorials, I've found that only two entries can be added using the format

`.. index:: main_entry; sub_entry`

However, I am looking for a method to include more than two entries, similar to LaTeX's approach with

`\index{main_entry!sub_entry!sub_sub_entry}`

Is there a way to achieve this in Sphinx?

**Comment by AA-Turner** [comment]
Does the following work?

```rst
.. index:: main_entry; sub_entry1
           main_entry; sub_entry2
```

It would be helpful to have an example of what the LaTeX syntax generates in the index.

A

**Reply by tarcine** [reply]
![Screenshot from 2024-11-06 09-08-30](https://github.com/user-attachments/assets/102b2805-bb8a-48c2-8e84-ffe9c56146fa)
the code you suggest will create two subentries.  

What I'm looking for is to create multi-level indexing e.g. subsubentry->subentry->mainentry. The latex code gives
![Screenshot from 2024-11-06 09-15-55](https://github.com/user-attachments/assets/44269620-143e-4c17-a6c2-947c0f53555d)
